src-web/components/modals/AnsibleAutomationModal/AnsibleCredentialsSelection.js

```diff
@@ -61,7 +61,7 @@ export const renderAnsibleCredentialsSelection = ({
       {!ansCredentialFlag &&
         <div className='infoArea createCredential'>
           <Text>
-            {msgs.get('ansible.no.credential', locale)}
+            {`${msgs.get('ansible.no.credential', locale)} `}
```

Comment:
This is required exactly 8px by designer, I fixed this in another stylish pr by css so no need extra space here

Reply:
Done